packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -1632,3 +2130,335 @@ Specific property types support a subset of [JSON Schema validations](https://to
+
+```json
+{
+  "id": "https://${yourOktaDomain}/meta/schemas/group/default",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "name": "example group",
+  "title": "Okta group example",
+  "lastUpdated": "2017-07-18T22:37:55.000Z",
+  "description": "Okta group profile template",
+  "created": "2017-07-18T22:35:30.000Z",
+  "definitions": {
+    "base": {
+      "id": "#base",
+      "properties": {
+        "description": {
+          "description": "Description",
+          "master": {
+            "type": "PROFILE_MASTER"
+          },
+          "maxLength": 1024,
+          "mutability": "READ_WRITE",
+          "permissions": [
+            {
+              "action": "READ_WRITE",
+              "principal": "SELF"
+            }
+          ],
+          "scope": "NONE",
+          "title": "Description",
+          "type": "string"
+        },
+        "name": {
+          "description": "Name",
+          "master": {
+            "type": "PROFILE_MASTER"
+          },
+          "maxLength": 255,
+          "mutability": "READ_WRITE",
+          "permissions": [
+            {
+              "action": "READ_WRITE",
+              "principal": "SELF"
+            }
+          ],
+          "required": true,
+          "scope": "NONE",
+          "title": "Name",
+          "type": "string"
+        }
+      },
+      "required": [
+        "name"
+      ],
+      "type": "object"
+    },
+    "custom": {
+      "id": "#custom",
+      "properties": {
+        "exampleCustomProperty": {
+          "description": "exampleCustomProperty",
+          "master": {
+            "type": "PROFILE_MASTER"
+          },
+          "maxLength": 20,
+          "minLength": 1,
+          "mutability": "READ_WRITE",
+          "permissions": [
+            {
+              "action": "READ_WRITE",
+              "principal": "SELF"
+            }
+          ],
+          "required": true,
+          "scope": "NONE",
+          "title": "exampleCustomProperty",
+          "type": "string",
+          "unique": "UNIQUE_VALIDATED"
+        }
+      },
+      "required": [
+        "exampleCustomProperty"
+      ],
+      "type": "object"
+    }
+  },
+  "type": "object",
+  "properties": {
+    "profile": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/base"
+        },
+        {
+          "$ref": "#/definitions/custom"
+        }
+      ]
+    }
+  },
+  "_links": {
+    "self": {
+      "href": "https://${yourOktaDomain}/api/v1/meta/schemas/group/default",
+      "method": "GET",
+      "rel": "self"
+    }
+  }
+}
+```
+
+### Schema properties
+
+The Group Schema is a valid [JSON Schema Draft 4](https://tools.ietf.org/html/draft-zyp-json-schema-04) document with the following properties:
+
+| Property    | Description                                                                              | DataType                                                           | Nullable | Unique | Readonly | Validation  |
```

Comment:
Readonly > Read-only

Reply:
I think Engineering is using this heading their property table templates. We should address this with the group.